language/README.md

```diff
@@ -31,30 +31,30 @@ Learning API.
 
 View the [documentation][analyze_docs] or the [source code][analyze_code].
 
-__Usage:__ `node analyze --help`
+__Usage:__ `node analyze.js --help`
 
 ```
 Commands:
-  sentimentOfText <text>               Detect the sentiment of a block of text.
-  sentimentInFile <bucket> <filename>  Detect the sentiment of text in a GCS file.
-  entitiesOfText <text>                Detect the entities of a block of text.
-  entitiesInFile <bucket> <filename>   Detect the entities of text in a GCS file.
-  syntaxOfText <text>                  Detect the syntax of a block of text.
-  syntaxInFile <bucket> <filename>     Detect the syntax of text in a GCS file.
+  sentiment-text <text>               Detects sentiment of a string.
+  sentiment-file <bucket> <filename>  Detects the sentiment in a file in Cloud Storage.
```

Comment:
Its not a specific group of sentiments there no "the" here. Simply "Detects sentiment in a file..."